client/components/navbar/navbar-notification.service.js

```diff
@@ -0,0 +1,105 @@
+class NavbarNotificationService {
```

Comment:
It looks like this code is duplicated in all 4 managers, right ? Maybe it should be a component in its own repository

Reply:
Yeah, in fact it's already a component, this is the service that handle the configuration of it.

Reply:
So is all the code for the Navbar and Sidebar, it has already been determined it was a faster and less elegant approche.

Reply:
It cannot be the same, translator and constants are not the same across all managers.